extkeys/mnemonic_test.go

```diff
@@ -18,16 +16,54 @@ type Vector struct {
 	language, salt, password, input, mnemonic, seed, xprv string
 }
 
+func TestNewMnemonic(t *testing.T) {
+	m1 := NewMnemonic("")
+	if m1.salt != Salt {
+		t.Errorf("expected default salt, got: %q", m1.salt)
+	}
+
+	customSalt := "custom-salt"
+	m2 := NewMnemonic(customSalt)
+	if m2.salt != customSalt {
+		t.Errorf("expected %q, got: %q", customSalt, m2.salt)
+	}
+}
+
+func TestMnemonic_WordList(t *testing.T) {
+	m := NewMnemonic("")
```

Comment:
You can use default salt (`NewMnemonic(Salt)`) if you test a regular case. Let's the most explicit usage in tests.